subsys/net/lib/lwm2m/Kconfig.ucifi

```diff
@@ -25,4 +25,87 @@ config LWM2M_UCIFI_BATTERY_INSTANCE_COUNT
 
 endif # LWM2M_UCIFI_BATTERY
 
+menuconfig LWM2M_UCIFI_LPWAN
+	bool "uCIFI LPWAN (3412) support"
+	help
+	  Include support for LWM2M Connectivity Monitoring Object (ID 4)
```

Comment:
Mismatch of help text and object name. This is not a Connectivity Monitoring object.

Reply:
It has now been fixed.
Thanks